tangelo/linq/tests/test_simulator.py

```diff
@@ -139,6 +139,20 @@ def test_simulate_mixed_state_save_measures(self):
             assert_freq_dict_almost_equal(sim.all_frequencies, reference_all, 8e-2)
             assert_freq_dict_almost_equal(sim.mid_circuit_meas_freqs, reference_mid, 8e-2)
 
+    def test_simulate_mixed_state_desired_state(self):
+        """ Test mid-circuit measurement (mixed-state simulation) for compatible/testable formats and backends.
+        Mixed-state do not have a statevector representation, as they are a statistical mixture of several statevectors.
+        Simulating individual shots is suitable.
```

Comment:
Mixed-state -> Mixed states, they are a statistical mixture -> they are statistical mixtures
+
What does mean suitable in that case? Is it possible but not recommended?

Reply:
(same comments for below)

Reply:
Updated the docstring to be in line with the previous test